tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/PersonValidator.java

```diff
@@ -133,7 +135,8 @@ private List<FieldError> validateForBulk(PersonDTO personDto) {
     fieldErrors.addAll(gdcDetailsValidator.validateForBulk(personDto.getGdcDetails()));
     fieldErrors.addAll(gmcDetailsValidator.validateForBulk(personDto.getGmcDetails()));
     fieldErrors.addAll(personalDetailsValidator.validateForBulk(personDto.getPersonalDetails()));
-    fieldErrors.addAll(rightToWorkValidator.validateForBulk(personDto.getRightToWork()));
+    fieldErrors.addAll(rightToWorkValidator.validateForBulk(personDto.getRightToWork(),
+        personDto));
```

Comment:
passing personDto.getRightToWork() and personDto feels a bit redundant.  Almost as if it might be valid to pass a RightToWorkDto that isn't part of the PersonDto.
I'd recommend just passing the personDto.  Another option would be to pass `personDto.getId()` as it looks like the only field used from outside the RightToWorkDto.